src/client/datascience/jupyter/jupyterExecution.ts

```diff
@@ -118,15 +118,15 @@ export class JupyterExecutionBase implements IJupyterExecution {
 
     /* eslint-disable complexity,  */
     public connectToNotebookServer(
-        options?: INotebookServerOptions,
```

Comment:
Found this was never optional, we always pass values.
Found a large number of places where this was the case, but couldn't be bothered fixing that.
Came here when looking at `displayUI`